tests/Makefile

```diff
@@ -112,9 +112,12 @@ BINARIES := $(addsuffix $(EXEXT),$(APPS))
 
 .SILENT:
 
-.PHONY: all check test clean
+.PHONY: all check test clean fuzz
 
-all: $(BINARIES)
+all: $(BINARIES) fuzz
+
```

Comment:
Please add a `.gitignore` for generated files in `tests/fuzz`. You can use this commit: https://github.com/gilles-peskine-arm/mbedtls/commit/f1fabeccffccaa2bc66a7bd860dca8646c4d60ac